src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
+
+    if (property.length > 0) {
```

Comment:
The check for `property.length > 0` is good for ensuring that empty properties are not added. However, you should also ensure that `value` is not undefined or empty before adding the property-value pair to the object.